install/error-handling.sh

```diff
@@ -19,26 +19,62 @@ function send_event {
   command popd > /dev/null
 }
 
-if [[ -f ../.reporterrors ]]; then
-  if [[ "$(cat ../.reporterrors)" == "yes" ]]; then
+reporterrors="$basedir/.reporterrors"
```

Comment:
ah yeah, I forgot we defined basedir. It might be helpful to change variables we expect used elsewhere to follow UPPERCASE, which I think is the common convention (but that can be another PR).

Reply:
Could do, not sure how consistent we are. I think of `UPPERCASE` as standard especially for user-facing envvars, `basedir` is internal to the installer.